framework/src/st_things/things_stack/things_types.h

```diff
@@ -38,6 +38,7 @@
 #define MAX_RT_CNT            10
 
 #define MAX_SSID_LEN          33
+#define MAX_BSSID_LEN         30
```

Comment:
wifi_manager's bssis lenth is 18
please check

i think remove this define after that.
using define by wifi_manager.h
it's better than redefine.

Reply:
@harish-191 Sorry for my previous comment.
The MAX_BSSID_LEN should be 17 without referenct to ipv4 or ipv6.

Reply:
@JeongJunSik  fixed.